src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -42,28 +48,23 @@ public void adsRecoveryPageFairOasis(Page page) {
   }
 
   @Test(
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsRecoveryPageFairOasis",
       groups = "AdsRecoveryNoAdblockPageFairOasis"
   )
-  public void adsRecoveryNoAdblockPageFairOasis(Page page) {
-    String url = urlBuilder.getUrlForPage(page);
+  public void adsRecoveryNoAdblockPageFairOasis() {
+    System.out.println(RECOVERY_ARTICLE);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)